src/Behat/Behat/ApplicationFactory.php

```diff
@@ -126,12 +126,16 @@ protected function getConfigPath()
             $cwd . 'behat.yml.dist',
             $cwd . 'behat.dist.yaml',
             $cwd . 'behat.dist.yml',
+            $cwd . 'behat.php',
+            $cwd . 'behat.php.dist',
```

Comment:
Should also add the `behat.dist.php` possibilty

Reply:
I'd actually propose `behat.dist.php` **instead of** `behat.php.dist`. We mostly support `behat.yml.dist` for compatibility reasons, but if the goal is to get IDE highlighting and tool support, forcing people to use a recognisable `.php` final extension is reasonable IMO

Reply:
With Yaml we support all three possibilities, with the .dist.yml listed last, I was suggesting to do the same here, support all three options, with .dist.php listed last, to keep it similar to Yaml

Reply:
I guess I just think there's limited value to supporting `.php.dist` as a new thing (other than parity with YAML) and it would reduce the complexity and number of cases to test? But I'm not strongly opposed to supporting all three.

Reply:
From userland point of view, I don't think any reason to add .dist.php support.
I think it would be better to deprecate dist.yml instead (in a future PR). Moreover, maybe deprecate "yml" extension instead of "yaml" too.

Reply:
.dist.yml was added on purpose, see the PR https://github.com/Behat/Behat/pull/1464

Reply:
hum ok.... so let's go

Reply:
done

Reply:
I don't think we need to add `behat.php.dist`.
`behat.dist.php` is strictly superior, by providing PHP syntax highlighting by default in any IDE (and in GitHub/Gitlab), while `.php.dist` would not without a special configuration of the IDE (a,d asking the GitHub team to add such special case in their product).

We cannot remove support for `behat.yml.dist` for BC reasons (and even the deprecation is probably not worth the cost for the ecosystem, compared to the maintenance cost it has), but I think we should avoid adding _new_ `*.dist` options.

Reply:
@stof mmm, ok, you may be right. @loic425 sorry for the noise, can you please remove the `.php.dist` option?Thanks again

Reply:
@carlos-granados agree with @stof ?
That makes a lot of sense to me.

Reply:
done https://github.com/Behat/Behat/pull/1537/commits/5fa5aabc265a69cffc9b798b54bb8baeb43d445a